aldor/aldor/lib/java/src/foamj/Foam.java

```diff
@@ -96,7 +96,7 @@ public static Object throwException(RuntimeException e) {
 	}
 
     public static int ptrToSInt(Object o) {
-		throw new RuntimeException();
+	return System.identityHashCode(o);
 	}
```

Comment:
It might make sense to use an auto-formatter on java code at some point, to avoid mixing tabs and spaces.